src/test/select-element-creation-strategy.test.ts

```diff
@@ -1,47 +1,46 @@
 const {SelectElementCreationStrategy} = require('../form-element-creator/select-element-creation-strategy')
 
-test('create select element with correct label', () => {
-    // given
-    const propertiesConfig = {
-        selects: [{
-            class: 'selects',
-            type: 'radio',
-            name: 'gender',
-            for: 'Female'
-        }]
-    };
-    const expectedHTMLInput = '<input type="radio" name="gender" placeholder="Female">';
-    const expectedHTMLLabel = '<label>Female</label>';
-
-    // when
-    const element = new SelectElementCreationStrategy().create(propertiesConfig.selects[0]);
-
-    // then
-    expect(element.outerHTML).toEqual(expectedHTMLLabel);
-    expect(element.outerHTML).toEqual(expectedHTMLInput);
-
-    expect(document.querySelector('.input-data-gender')).toBeDefined();
-});
-
-test('create select element with correct label', () => {
-    // given
-    const propertiesConfig = {
-        selects: [{
-            class: 'selects',
-            type: 'radio',
-            name: 'gender',
-            for: 'Male'
-        }]
-    };
-    const expectedHTMLInput = '<input type="radio" name="gender" placeholder="Male">';
-    const expectedHTMLLabel = '<label>Male</label>';
-
-    // when
-    const element = new SelectElementCreationStrategy().create(propertiesConfig.selects[0]);
-
-    // then
-    expect(element.outerHTML).toEqual(expectedHTMLLabel);
-    expect(element.outerHTML).toEqual(expectedHTMLInput);
-
-    expect(document.querySelector('.input-data-gender')).toBeDefined();
-});
+describe('SelectElementCreationStrategy unit tests', () => {
+    it('create select element with correct label', () => {
+        // given
+        const propertiesConfig = {
+            selects: [{
+                class: 'selects',
+                type: 'radio',
+                name: 'gender',
+                for: 'Female',
+                classNameLabel: 'gender-label'
+            }]
+        };
+        const expectedHTMLInput = '<input type="radio" name="gender">';
+        const expectedHTMLLabel = '<label class="gender-label">Female</label>';
+        // when
+        const elementInput = new SelectElementCreationStrategy().createInput(propertiesConfig.selects[0]);
```

Comment:
tu tez ten [0] mozna do zmiennej wyciagnac bo nie ma potrzeby wyciagac z tablicy tego samego elementu dwa razy. Poza tym to duplikacja kodu zamiast miec to w zmiennej. Tak samo popraw wszystkie inne miejsca ktore robia podobny zabieg